README.md

```diff
@@ -1,16 +1,34 @@
-# Title
+# Ruby Basics Lab
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Define the body of each method
```

Comment:
I think our LG here is "define implementation based on a test and a provided method signature"


based on...the signature? Based on a test? something like that.

This LG is kinda unique to this lab, i think it should be made more general